services/housefy/src/main/java/com/dech/housefy/domain/RoleDomain.java

```diff
@@ -0,0 +1,17 @@
+package com.dech.housefy.domain;
+
+import java.util.List;
+
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import lombok.Data;
+
+@Data
+@Document(collection = "role")
+public class RoleDomain {
```

Comment:
Since that we are using just the name of the class in the whole project please change it to Role.java instead of RoleDomain.java or UserRole or something like that, maybe in the future we can discuss about the coding standards an follow some patterns.